### PR TITLE
Feature/bars per hour

### PR DESCRIPTION
**Pull request by hugocowan**
This adds a bars/hr statistic to the overlay. Over time it becomes quite accurate and it's a nice stat to be able to see.

### CODE REVIEW

File: src/main/java/com/toofifty/easyblastfurnace/utils/SessionStatistics.java [hunk 1]
```diff
@@ -31,6 +33,11 @@
     @Getter
     private int staminaDoses = 0;
 
+    private Instant timeStarted;
+
+    @Getter
+    private long barsPerHour = 0;
```
Comment (Toofifty) [inline src/main/java/com/toofifty/easyblastfurnace/utils/SessionStatistics.java:39]: `long`? How many bars do you think people can make per hour? 😂

**Review by Toofifty — APPROVED** [review]
LGTM - you mentioned you may have a few more PRs, shall I wait for those before pushing an update?

File: src/main/java/com/toofifty/easyblastfurnace/utils/SessionStatistics.java (hunk 1 above)
Comment (hugocowan) [reply src/main/java/com/toofifty/easyblastfurnace/utils/SessionStatistics.java]: Good point lol

**Comment by hugocowan** [comment]
I think it'd be fine to merge but yeah, maybe wait on the next PR before pushing the update.

The next one is dependent on this PR getting merged, which has taken a lot longer than I'd hoped for https://github.com/runelite/runelite/pull/15621

**Pull request merged**